build.sbt

```diff
@@ -13,24 +16,48 @@ val headerMsg =  """/** This software is released under the University of Illino
                         |  */
                         |""".stripMargin
 
-lazy val root = (project in file(".")).
-  aggregate(saulCore, saulExamples)
-  .enablePlugins(AutomateHeaderPlugin)
+
+lazy val saulUser = System.getenv("SAUL_USER")
+lazy val user = if(saulUser == null) System.getProperty("user.name") else saulUser
```

Comment:
Actually I meant `if (saulUser` right after the keyword. Won't be that picky though.